### PR TITLE
Guest formatting

### PR DESCRIPTION
**Pull request by lboje**
Some formatting stuff! :)

Going to work on fonts and such next

RSVP: attending

desktop
image
image

mobile
image
image

RSVP: declined
desktop:
image

mobile:
image

Bonus: not signed in
image
image

code entry
image

### CODE REVIEW

File: client/src/app/components/forms/code-input-form/code-input-form.component.scss [hunk 1]
```diff
@@ -25,7 +25,7 @@
 	font-size: 2em;
 	text-align: center;
 	padding-bottom: 5px;
-	font-family: auto;
+	font-family: Spectral;
```
Comment (lvoytek) [inline client/src/app/components/forms/code-input-form/code-input-form.component.scss:28]: Good practice to add a backup font here even though we know its available. Something like:

`font-family: Spectral serif`

File: client/src/app/components/forms/rsvp/rsvp.component.scss [hunk 2]
```diff
@@ -1,4 +1,43 @@
+@import '../../../../styles.scss';
+
+hr {
+	//TODO: Figure out how to pull in ll-green 800 :/
```
Comment (lvoytek) [inline client/src/app/components/forms/rsvp/rsvp.component.scss:4]: Does importing wedding-theme.scss fail?

File: client/src/assets/fonts/OFL__Spectral.txt
Comment (lvoytek) [inline client/src/assets/fonts/OFL__Spectral.txt:1]: The filename here has 2 underscores

**Review by lvoytek — APPROVED** [review]
Nice! Minor change recommendations added

File: client/src/app/components/forms/rsvp/rsvp.component.scss (hunk 2 above)
Comment (lboje) [reply client/src/app/components/forms/rsvp/rsvp.component.scss:4]: it's much more...weird. technically, no - you have to be pretty picky about it.

but you can't actually get the colors from there! for some fucking reason? it just throws several hissy fits no matter how you go about it - accessing the color from within the palette fails.

File: client/src/assets/fonts/OFL__Spectral.txt
Comment (lboje) [reply client/src/assets/fonts/OFL__Spectral.txt:1]: fixed

File: client/src/app/components/forms/rsvp/rsvp.component.scss (hunk 2 above)
Comment (lvoytek) [reply client/src/app/components/forms/rsvp/rsvp.component.scss:4]: Oof, we can look into that later then

File: client/src/app/components/forms/code-input-form/code-input-form.component.scss (hunk 1 above)
Comment (lboje) [reply client/src/app/components/forms/code-input-form/code-input-form.component.scss:28]: would serif be the backup font here? or

Comment (lboje) [reply client/src/app/components/forms/code-input-form/code-input-form.component.scss]: pushed up

**Pull request merged**